ui/src/app/utils/title.ts

```diff
@@ -0,0 +1,14 @@
+import {environment} from 'environments/environment';
+
+import {BASE_TITLE} from 'app/utils/strings';
+
+export function buildPageTitleForEnvironment(pageTitle?: string) {
+  let title = BASE_TITLE;
+  if (environment.shouldShowDisplayTag) {
+    title = `[${environment.displayTag}] ` + title;
```

Comment:
nit: avoid concatenating with interpolation:  
```suggestion
    title = `[${environment.displayTag}] title` ;
```